### ISSUE TITLE
[rmodels] Improved 3D animation performance

### ISSUE THREAD

**Issue by WOLFxxxxxx** [raysan5/raylib#4587]
![Screenshot_٢٠٢٤-١٢-٠٩-١٦-٥٧-٤٦-٤٠١_com android chrome-edit](https://github.com/user-attachments/assets/30142011-eb14-4bf5-88a1-7d5d87126f4c)
3D animation needs to improve performance or make it run on a GPU instead of a CPU. It is not possible to run more than 20 animated characters.

**Comment by Volcanic-Penguin** [comment]
I 100% agree that it should be easier to target the dedicated GPU over the integrated GPU. I fixed this for myself by changing a variable in the C source code. Find where the variable GLFW_USE_HYBRID_HPG is set to OFF and set it to ON. 

`cmake_dependent_option(GLFW_USE_HYBRID_HPG "Force use of high-performance GPU on hybrid systems" OFF
`
Setting this to ON will trigger another place in the code to set the following variables:

```
// Executables (but not DLLs) exporting this symbol with this value will be
// automatically directed to the high-performance GPU on Nvidia Optimus systems
// with up-to-date drivers
//
__declspec(dllexport) DWORD NvOptimusEnablement = 1;

// Executables (but not DLLs) exporting this symbol with this value will be
// automatically directed to the high-performance GPU on AMD PowerXpress systems
// with up-to-date drivers
//
__declspec(dllexport) int AmdPowerXpressRequestHighPerformance = 1;

```

I've been considering posting an issue about this myself.

**Comment by raysan5** [comment]
GPU skinning was implemented recently in `raylib 5.5`, you can check the linked example for reference.
